content/docs/04-clusters/1-new-clusters/03-azure.md

```diff
@@ -84,15 +88,22 @@ The following steps need to be performed to provision a new Azure cluster:
     | **Region** | Select a region in Azure in which the cluster should be deployed.|
     | **Resource Group** | Select the resource group in which the cluster should be deployed.|
     | **SSH Key** | Public key to configure remote SSH access to the nodes.|
-    | **Static Placement** | By default, Palette uses dynamic placement, wherein a new VPC with a public and private subnet is created to place cluster resources for every cluster.<br /> These resources are fully managed by Palette and deleted when the corresponding cluster is deleted. Turn on the **Static Placement** option if it's desired to place resources into preexisting VPCs and subnets. If the user is making the selection of **Static Placement** of resources, the following placement information needs to be provided:
-    || **Virtual Network**: Select the virtual network from dropdown menu.
-    || **Control plane Subnet**: Select the control plane network from the dropdown menu.
-    || **Worker Network**: Select the worker network from the dropdown.

```

Comment:
In the normal flow you'd still have to select those.
If you select an cloud account with Disable properties enabled then you have to input the right details as described: 
```
Name <br /> CIDR Block <br /> Security Group Name
```